admin-site/src/App.svelte

```diff
@@ -13,11 +13,10 @@
   import EventsList from './EventsList.svelte';
   import RegistrationForm from './RegistrationForm.svelte'
   import DevTools from './DevTools.svelte';
+  import EventCreation from './EventCreation.svelte';
```

Comment:
`EventCreationForm` would be a bit more explicit

Reply:
Same below with OccasionCreation, it's missing a noun :)

Reply:
Implemented